src/containers/find-offer/OfferRequestBlock.jsx

```diff
@@ -0,0 +1,25 @@
+import { useTranslation } from 'react-i18next'
+import useBreakpoints from '~/hooks/use-breakpoints'
+
+import UpperBlock from '~/components/upper-block/UpperBlock'
+import Button from '@mui/material/Button'
+import icon from '~/assets/img/find-offer/subject_icon.png'
+
+const OfferRequestBlock = () => {
+  const { t } = useTranslation()
+  const { isMobile } = useBreakpoints()
+
+  return (
+    <UpperBlock img={ icon } path='findOffer.offerRequestBlock' >
```

Comment:
naming is not clear what means path? it is not used for link and move this string to consts

Reply:
fixed